### PR TITLE
Main window menu updates

### PR DESCRIPTION
**Pull request by roman4750**

### CODE REVIEW

**Comment by tomas-pluskal** [comment]
I would rename ListViewGroups to GroupableListView

File: src/main/java/io/github/mzmine/gui/mainwindow/MainWindowController.java [hunk 1]
```diff
@@ -693,19 +693,19 @@ public void addTab(Tab tab) {
     if (tab instanceof MZmineTab) {
       ((MZmineTab) tab).updateOnSelectionProperty().addListener(((obs, old, val) -> {
         if (val) {
-          if (!((MZmineTab) tab).getRawDataFiles()
+          if (((MZmineTab) tab).getRawDataFiles() != null && !((MZmineTab) tab).getRawDataFiles()
```
Comment (SteffenHeu) [inline src/main/java/io/github/mzmine/gui/mainwindow/MainWindowController.java:696]: Actually, this should never be null, as it should return an empty collection by default. 
Double checking is good, but the exception should not have happened.

Comment (roman4750) [reply src/main/java/io/github/mzmine/gui/mainwindow/MainWindowController.java:696]: Yeah, but RawDataOverviewPane getFeatureLists() doesn't override `@Nonnull` from MZmineTab, so it caused NPE:
```
  @Override
  public Collection<? extends FeatureList> getFeatureLists() {
    return null;
  }
```
I'm gonna fix it

**Comment by tomas-pluskal** [comment]
Looks good, thanks!

**Pull request merged**